Shared.EventStore/EventStore/EventStoreContext.cs

```diff
@@ -541,286 +222,22 @@ private void LogError(Exception exception)
-            {
-                // Try a DNS resolve here
-                IPAddress[] addresses = Dns.GetHostAddresses(this.EventStoreConnectionSettings.IpAddress);
-
-                if (addresses.Any())
-                {
-                    address = addresses.First();
-                }
-                else
-                {
-                    throw new ArgumentException($"Unable to get IP address for Event Store Settings Config Value [{this.EventStoreConnectionSettings.IpAddress}]");
-                }
-            }
-
-            IPEndPoint endpoint = new IPEndPoint(address, this.EventStoreConnectionSettings.HttpPort);
-
-            return new ProjectionsManager(new ConsoleLogger(), endpoint, TimeSpan.FromSeconds(30));
-        }
-
-        /// <summary>
-        /// Gets the event store connection.
-        /// </summary>
-        /// <returns></returns>
-        private async Task<IEventStoreConnection> GetEventStoreConnection()
-        {
-            if (this.Connection == null)
-            {
-                lock(this.CreateLockObject)
-                {
-                    if (this.Connection == null)
-                    {
-                        this.Connection = this.ConnectionResolver(this.EventStoreConnectionSettings);
-                        this.Connection.Connected += this.Connection_Connected;
-                        this.Connection.Reconnecting += this.Connection_Reconnecting;
-                        this.Connection.Closed += this.Connection_Closed;
-                        this.Connection.ErrorOccurred += this.Connection_ErrorOccurred;
-                        this.Connection.Disconnected += this.Connection_Disconnected;
-                        this.Connection.AuthenticationFailed += this.Connection_AuthenticationFailed;
-                        this.Connection.ConnectAsync().Wait();
-                    }
-                }
-            }
-
-            return this.Connection;
-        }
-
-        /// <summary>
-        /// Persistents the subscription dropped.
+        /// Traces the specified trace.
         /// </summary>
-        /// <param name="streamName">Name of the stream.</param>
-        /// <param name="groupName">Name of the group.</param>
-        /// <param name="subscriptionDropReason">The subscription drop reason.</param>
-        /// <param name="exception">The exception.</param>
-        /// <param name="subscriptionGroupId">The subscription group identifier.</param>
-        private void PersistentSubscriptionDropped(String streamName,
-                                                   String groupName,
-                                                   SubscriptionDropReason subscriptionDropReason,
-                                                   Exception exception,
-                                                   Guid subscriptionGroupId)
+        /// <param name="trace">The trace.</param>
+        private void LogInformation(String trace)
```

Comment:
Codacy found an issue: [Remove the unused private method 'LogInformation'.](https://app.codacy.com/manual/stuart_ferguson1/Shared/pullRequest?prid=5765119)